<html><head><meta content="text/html; charset=UTF-8" http-equiv="content-type"><style type="text/css">.lst-kix_276sblcrra50-3>li{counter-increment:lst-ctn-kix_276sblcrra50-3}ol.lst-kix_58t3yr50d134-4.start{counter-reset:lst-ctn-kix_58t3yr50d134-4 0}ol.lst-kix_276sblcrra50-2.start{counter-reset:lst-ctn-kix_276sblcrra50-2 0}.lst-kix_58t3yr50d134-7>li{counter-increment:lst-ctn-kix_58t3yr50d134-7}ol.lst-kix_9w6729rw7kqq-6.start{counter-reset:lst-ctn-kix_9w6729rw7kqq-6 0}.lst-kix_9w6729rw7kqq-0>li{counter-increment:lst-ctn-kix_9w6729rw7kqq-0}ol.lst-kix_58t3yr50d134-7.start{counter-reset:lst-ctn-kix_58t3yr50d134-7 0}.lst-kix_9w6729rw7kqq-1>li{counter-increment:lst-ctn-kix_9w6729rw7kqq-1}.lst-kix_276sblcrra50-4>li{counter-increment:lst-ctn-kix_276sblcrra50-4}.lst-kix_58t3yr50d134-6>li{counter-increment:lst-ctn-kix_58t3yr50d134-6}ol.lst-kix_276sblcrra50-5.start{counter-reset:lst-ctn-kix_276sblcrra50-5 0}.lst-kix_58t3yr50d134-8>li:before{content:"" counter(lst-ctn-kix_58t3yr50d134-8,lower-roman) ". "}.lst-kix_58t3yr50d134-5>li:before{content:"" counter(lst-ctn-kix_58t3yr50d134-5,lower-roman) ". "}.lst-kix_58t3yr50d134-6>li:before{content:"" counter(lst-ctn-kix_58t3yr50d134-6,decimal) ". "}.lst-kix_58t3yr50d134-3>li:before{content:"" counter(lst-ctn-kix_58t3yr50d134-3,decimal) ". "}.lst-kix_58t3yr50d134-7>li:before{content:"" counter(lst-ctn-kix_58t3yr50d134-7,lower-latin) ". "}ol.lst-kix_y7q18invd69n-4.start{counter-reset:lst-ctn-kix_y7q18invd69n-4 0}.lst-kix_y7q18invd69n-0>li{counter-increment:lst-ctn-kix_y7q18invd69n-0}.lst-kix_58t3yr50d134-4>li:before{content:"" counter(lst-ctn-kix_58t3yr50d134-4,lower-latin) ". "}ol.lst-kix_58t3yr50d134-1.start{counter-reset:lst-ctn-kix_58t3yr50d134-1 0}.lst-kix_58t3yr50d134-8>li{counter-increment:lst-ctn-kix_58t3yr50d134-8}.lst-kix_58t3yr50d134-1>li:before{content:"" counter(lst-ctn-kix_58t3yr50d134-1,lower-latin) ". "}.lst-kix_58t3yr50d134-2>li:before{content:"" counter(lst-ctn-kix_58t3yr50d134-2,lower-roman) ". "}.lst-kix_58t3yr50d134-0>li:before{content:"" counter(lst-ctn-kix_58t3yr50d134-0,decimal) ". "}ol.lst-kix_y7q18invd69n-1.start{counter-reset:lst-ctn-kix_y7q18invd69n-1 0}.lst-kix_9w6729rw7kqq-3>li{counter-increment:lst-ctn-kix_9w6729rw7kqq-3}.lst-kix_y7q18invd69n-2>li{counter-increment:lst-ctn-kix_y7q18invd69n-2}.lst-kix_y7q18invd69n-8>li{counter-increment:lst-ctn-kix_y7q18invd69n-8}.lst-kix_276sblcrra50-0>li{counter-increment:lst-ctn-kix_276sblcrra50-0}ol.lst-kix_y7q18invd69n-0.start{counter-reset:lst-ctn-kix_y7q18invd69n-0 0}ol.lst-kix_y7q18invd69n-7.start{counter-reset:lst-ctn-kix_y7q18invd69n-7 0}ol.lst-kix_276sblcrra50-7.start{counter-reset:lst-ctn-kix_276sblcrra50-7 0}.lst-kix_58t3yr50d134-5>li{counter-increment:lst-ctn-kix_58t3yr50d134-5}ol.lst-kix_9w6729rw7kqq-1.start{counter-reset:lst-ctn-kix_9w6729rw7kqq-1 0}.lst-kix_276sblcrra50-1>li{counter-increment:lst-ctn-kix_276sblcrra50-1}.lst-kix_y7q18invd69n-3>li{counter-increment:lst-ctn-kix_y7q18invd69n-3}ol.lst-kix_276sblcrra50-0.start{counter-reset:lst-ctn-kix_276sblcrra50-0 0}.lst-kix_9w6729rw7kqq-4>li{counter-increment:lst-ctn-kix_9w6729rw7kqq-4}ol.lst-kix_y7q18invd69n-6.start{counter-reset:lst-ctn-kix_y7q18invd69n-6 0}ol.lst-kix_276sblcrra50-1.start{counter-reset:lst-ctn-kix_276sblcrra50-1 0}ol.lst-kix_276sblcrra50-8.start{counter-reset:lst-ctn-kix_276sblcrra50-8 0}.lst-kix_276sblcrra50-7>li{counter-increment:lst-ctn-kix_276sblcrra50-7}ol.lst-kix_9w6729rw7kqq-7.start{counter-reset:lst-ctn-kix_9w6729rw7kqq-7 0}ol.lst-kix_58t3yr50d134-0.start{counter-reset:lst-ctn-kix_58t3yr50d134-0 0}ol.lst-kix_58t3yr50d134-6.start{counter-reset:lst-ctn-kix_58t3yr50d134-6 0}ol.lst-kix_9w6729rw7kqq-0.start{counter-reset:lst-ctn-kix_9w6729rw7kqq-0 0}.lst-kix_9w6729rw7kqq-6>li{counter-increment:lst-ctn-kix_9w6729rw7kqq-6}ol.lst-kix_58t3yr50d134-4{list-style-type:none}ol.lst-kix_58t3yr50d134-3{list-style-type:none}ol.lst-kix_58t3yr50d134-6{list-style-type:none}ol.lst-kix_58t3yr50d134-5{list-style-type:none}.lst-kix_58t3yr50d134-2>li{counter-increment:lst-ctn-kix_58t3yr50d134-2}ol.lst-kix_58t3yr50d134-8{list-style-type:none}ol.lst-kix_58t3yr50d134-7{list-style-type:none}.lst-kix_y7q18invd69n-5>li{counter-increment:lst-ctn-kix_y7q18invd69n-5}.lst-kix_9w6729rw7kqq-7>li{counter-increment:lst-ctn-kix_9w6729rw7kqq-7}ol.lst-kix_y7q18invd69n-2.start{counter-reset:lst-ctn-kix_y7q18invd69n-2 0}.lst-kix_276sblcrra50-8>li{counter-increment:lst-ctn-kix_276sblcrra50-8}.lst-kix_y7q18invd69n-6>li{counter-increment:lst-ctn-kix_y7q18invd69n-6}ol.lst-kix_y7q18invd69n-5.start{counter-reset:lst-ctn-kix_y7q18invd69n-5 0}.lst-kix_y7q18invd69n-3>li:before{content:"" counter(lst-ctn-kix_y7q18invd69n-3,decimal) ". "}ol.lst-kix_9w6729rw7kqq-5.start{counter-reset:lst-ctn-kix_9w6729rw7kqq-5 0}.lst-kix_58t3yr50d134-0>li{counter-increment:lst-ctn-kix_58t3yr50d134-0}.lst-kix_y7q18invd69n-1>li:before{content:"" counter(lst-ctn-kix_y7q18invd69n-1,lower-latin) ". "}.lst-kix_y7q18invd69n-5>li:before{content:"" counter(lst-ctn-kix_y7q18invd69n-5,lower-roman) ". "}.lst-kix_y7q18invd69n-0>li:before{content:"" counter(lst-ctn-kix_y7q18invd69n-0,decimal) ". "}.lst-kix_y7q18invd69n-4>li:before{content:"" counter(lst-ctn-kix_y7q18invd69n-4,lower-latin) ". "}.lst-kix_58t3yr50d134-3>li{counter-increment:lst-ctn-kix_58t3yr50d134-3}ol.lst-kix_9w6729rw7kqq-7{list-style-type:none}ol.lst-kix_276sblcrra50-6.start{counter-reset:lst-ctn-kix_276sblcrra50-6 0}.lst-kix_y7q18invd69n-7>li:before{content:"" counter(lst-ctn-kix_y7q18invd69n-7,lower-latin) ". "}ol.lst-kix_9w6729rw7kqq-6{list-style-type:none}ol.lst-kix_9w6729rw7kqq-5{list-style-type:none}ol.lst-kix_9w6729rw7kqq-4{list-style-type:none}ol.lst-kix_276sblcrra50-3.start{counter-reset:lst-ctn-kix_276sblcrra50-3 0}ol.lst-kix_9w6729rw7kqq-3{list-style-type:none}ol.lst-kix_9w6729rw7kqq-2{list-style-type:none}ol.lst-kix_y7q18invd69n-8.start{counter-reset:lst-ctn-kix_y7q18invd69n-8 0}ol.lst-kix_58t3yr50d134-5.start{counter-reset:lst-ctn-kix_58t3yr50d134-5 0}ol.lst-kix_9w6729rw7kqq-1{list-style-type:none}.lst-kix_y7q18invd69n-6>li:before{content:"" counter(lst-ctn-kix_y7q18invd69n-6,decimal) ". "}ol.lst-kix_9w6729rw7kqq-0{list-style-type:none}ol.lst-kix_9w6729rw7kqq-2.start{counter-reset:lst-ctn-kix_9w6729rw7kqq-2 0}ol.lst-kix_9w6729rw7kqq-8.start{counter-reset:lst-ctn-kix_9w6729rw7kqq-8 0}ol.lst-kix_y7q18invd69n-0{list-style-type:none}ol.lst-kix_58t3yr50d134-0{list-style-type:none}ol.lst-kix_58t3yr50d134-2{list-style-type:none}ol.lst-kix_58t3yr50d134-1{list-style-type:none}ol.lst-kix_58t3yr50d134-8.start{counter-reset:lst-ctn-kix_58t3yr50d134-8 0}ol.lst-kix_y7q18invd69n-6{list-style-type:none}ol.lst-kix_y7q18invd69n-5{list-style-type:none}ol.lst-kix_y7q18invd69n-8{list-style-type:none}ol.lst-kix_y7q18invd69n-7{list-style-type:none}ol.lst-kix_y7q18invd69n-2{list-style-type:none}ol.lst-kix_y7q18invd69n-1{list-style-type:none}.lst-kix_y7q18invd69n-2>li:before{content:"" counter(lst-ctn-kix_y7q18invd69n-2,lower-roman) ". "}ol.lst-kix_y7q18invd69n-4{list-style-type:none}ol.lst-kix_y7q18invd69n-3{list-style-type:none}.lst-kix_276sblcrra50-6>li{counter-increment:lst-ctn-kix_276sblcrra50-6}ol.lst-kix_276sblcrra50-0{list-style-type:none}ol.lst-kix_276sblcrra50-1{list-style-type:none}.lst-kix_9w6729rw7kqq-1>li:before{content:"" counter(lst-ctn-kix_9w6729rw7kqq-1,lower-latin) ". "}.lst-kix_9w6729rw7kqq-2>li:before{content:"" counter(lst-ctn-kix_9w6729rw7kqq-2,lower-roman) ". "}.lst-kix_276sblcrra50-5>li{counter-increment:lst-ctn-kix_276sblcrra50-5}.lst-kix_y7q18invd69n-8>li:before{content:"" counter(lst-ctn-kix_y7q18invd69n-8,lower-roman) ". "}ol.lst-kix_9w6729rw7kqq-8{list-style-type:none}.lst-kix_9w6729rw7kqq-0>li:before{content:"" counter(lst-ctn-kix_9w6729rw7kqq-0,decimal) ". "}ol.lst-kix_58t3yr50d134-2.start{counter-reset:lst-ctn-kix_58t3yr50d134-2 0}ol.lst-kix_9w6729rw7kqq-4.start{counter-reset:lst-ctn-kix_9w6729rw7kqq-4 0}.lst-kix_276sblcrra50-8>li:before{content:"" counter(lst-ctn-kix_276sblcrra50-8,lower-roman) ". "}.lst-kix_276sblcrra50-7>li:before{content:"" counter(lst-ctn-kix_276sblcrra50-7,lower-latin) ". "}.lst-kix_9w6729rw7kqq-2>li{counter-increment:lst-ctn-kix_9w6729rw7kqq-2}.lst-kix_276sblcrra50-4>li:before{content:"" counter(lst-ctn-kix_276sblcrra50-4,lower-latin) ". "}ol.lst-kix_9w6729rw7kqq-3.start{counter-reset:lst-ctn-kix_9w6729rw7kqq-3 0}.lst-kix_9w6729rw7kqq-5>li{counter-increment:lst-ctn-kix_9w6729rw7kqq-5}.lst-kix_276sblcrra50-6>li:before{content:"" counter(lst-ctn-kix_276sblcrra50-6,decimal) ". "}li.li-bullet-0:before{margin-left:-18pt;white-space:nowrap;display:inline-block;min-width:18pt}.lst-kix_9w6729rw7kqq-8>li{counter-increment:lst-ctn-kix_9w6729rw7kqq-8}.lst-kix_276sblcrra50-2>li{counter-increment:lst-ctn-kix_276sblcrra50-2}.lst-kix_276sblcrra50-5>li:before{content:"" counter(lst-ctn-kix_276sblcrra50-5,lower-roman) ". "}.lst-kix_9w6729rw7kqq-4>li:before{content:"" counter(lst-ctn-kix_9w6729rw7kqq-4,lower-latin) ". "}.lst-kix_276sblcrra50-0>li:before{content:"" counter(lst-ctn-kix_276sblcrra50-0,decimal) ". "}ol.lst-kix_y7q18invd69n-3.start{counter-reset:lst-ctn-kix_y7q18invd69n-3 0}.lst-kix_9w6729rw7kqq-3>li:before{content:"" counter(lst-ctn-kix_9w6729rw7kqq-3,decimal) ". "}.lst-kix_9w6729rw7kqq-5>li:before{content:"" counter(lst-ctn-kix_9w6729rw7kqq-5,lower-roman) ". "}ol.lst-kix_276sblcrra50-4.start{counter-reset:lst-ctn-kix_276sblcrra50-4 0}.lst-kix_y7q18invd69n-4>li{counter-increment:lst-ctn-kix_y7q18invd69n-4}.lst-kix_276sblcrra50-3>li:before{content:"" counter(lst-ctn-kix_276sblcrra50-3,decimal) ". "}.lst-kix_y7q18invd69n-7>li{counter-increment:lst-ctn-kix_y7q18invd69n-7}.lst-kix_9w6729rw7kqq-8>li:before{content:"" counter(lst-ctn-kix_9w6729rw7kqq-8,lower-roman) ". "}ol.lst-kix_276sblcrra50-4{list-style-type:none}.lst-kix_58t3yr50d134-4>li{counter-increment:lst-ctn-kix_58t3yr50d134-4}ol.lst-kix_276sblcrra50-5{list-style-type:none}.lst-kix_9w6729rw7kqq-7>li:before{content:"" counter(lst-ctn-kix_9w6729rw7kqq-7,lower-latin) ". "}ol.lst-kix_276sblcrra50-2{list-style-type:none}ol.lst-kix_276sblcrra50-3{list-style-type:none}ol.lst-kix_58t3yr50d134-3.start{counter-reset:lst-ctn-kix_58t3yr50d134-3 0}.lst-kix_9w6729rw7kqq-6>li:before{content:"" counter(lst-ctn-kix_9w6729rw7kqq-6,decimal) ". "}.lst-kix_276sblcrra50-2>li:before{content:"" counter(lst-ctn-kix_276sblcrra50-2,lower-roman) ". "}ol.lst-kix_276sblcrra50-8{list-style-type:none}.lst-kix_276sblcrra50-1>li:before{content:"" counter(lst-ctn-kix_276sblcrra50-1,lower-latin) ". "}ol.lst-kix_276sblcrra50-6{list-style-type:none}ol.lst-kix_276sblcrra50-7{list-style-type:none}.lst-kix_y7q18invd69n-1>li{counter-increment:lst-ctn-kix_y7q18invd69n-1}.lst-kix_58t3yr50d134-1>li{counter-increment:lst-ctn-kix_58t3yr50d134-1}ol{margin:0;padding:0}table td,table th{padding:0}.c2{padding-top:0pt;padding-bottom:0pt;line-height:1.15;orphans:2;widows:2;text-align:left;height:11pt}.c0{color:#000000;font-weight:400;text-decoration:none;vertical-align:baseline;font-size:11pt;font-family:"Times New Roman";font-style:normal}.c6{padding-top:0pt;padding-bottom:0pt;line-height:1.15;orphans:2;widows:2;text-align:left}.c3{-webkit-text-decoration-skip:none;color:#1155cc;font-weight:400;text-decoration:underline;text-decoration-skip-ink:none;font-family:"Times New Roman"}.c5{-webkit-text-decoration-skip:none;color:#000099;font-weight:400;text-decoration:underline;text-decoration-skip-ink:none;font-family:"Times New Roman"}.c11{color:#000000;text-decoration:none;vertical-align:baseline;font-size:11pt;font-style:normal}.c18{-webkit-text-decoration-skip:none;color:#1155cc;text-decoration:underline;text-decoration-skip-ink:none}.c17{vertical-align:baseline;font-size:11pt;font-style:normal}.c15{background-color:#ffffff;max-width:468pt;padding:54pt 72pt 72pt 72pt}.c19{padding:0;margin:0}.c13{font-weight:400;font-family:"Arial"}.c7{font-weight:400;font-family:"Times New Roman"}.c1{font-weight:700;font-family:"Times New Roman"}.c8{margin-left:36pt;padding-left:0pt}.c9{color:inherit;text-decoration:inherit}.c14{margin-left:4.5pt}.c12{font-style:italic}.c4{margin-left:7pt}.c10{margin-left:6pt}.c16{text-indent:4.5pt}.title{padding-top:0pt;color:#000000;font-size:26pt;padding-bottom:3pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}.subtitle{padding-top:0pt;color:#666666;font-size:15pt;padding-bottom:16pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}li{color:#000000;font-size:11pt;font-family:"Arial"}p{margin:0;color:#000000;font-size:11pt;font-family:"Arial"}h1{padding-top:20pt;color:#000000;font-size:20pt;padding-bottom:6pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h2{padding-top:18pt;color:#000000;font-size:16pt;padding-bottom:6pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h3{padding-top:16pt;color:#434343;font-size:14pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h4{padding-top:14pt;color:#666666;font-size:12pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h5{padding-top:12pt;color:#666666;font-size:11pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h6{padding-top:12pt;color:#666666;font-size:11pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;font-style:italic;orphans:2;widows:2;text-align:left}</style></head><body class="c15 doc-content"><p class="c6"><span class="c1">Specific projects:</span></p><ol class="c19 lst-kix_276sblcrra50-0 start" start="1"><li class="c6 c8 li-bullet-0"><span class="c3"><a class="c9" href="#id.plu853aarphe">Bonobo genome compared with the chimpanzee and human genomes</a></span></li><li class="c6 c8 li-bullet-0"><span class="c3"><a class="c9" href="#id.g98eyemwl1o3">Bonobos fall within the genomic variation of chimpanzees</a></span><span class="c0">&nbsp;</span></li><li class="c6 c8 li-bullet-0"><span class="c3"><a class="c9" href="#id.lsv089d3trxa">Linkage disequilibrium extends across putative selected sites in FOXP2 </a></span></li><li class="c6 c8 li-bullet-0"><span class="c3"><a class="c9" href="#id.afc7sqbzs1o">Combining sperm typing and linkage disequilibrium analyses reveals differences in selective pressures or recombination rates across human populations</a></span></li><li class="c6 c8 li-bullet-0"><span class="c3"><a class="c9" href="#id.vnwo1z9n6imo">Analysis of one million base pairs of Neanderthal DNA</a></span></li><li class="c6 c8 li-bullet-0"><span class="c3"><a class="c9" href="#id.ly189d50cg7p">Why do human diversity levels vary at a megabase scale?</a></span></li><li class="c6 c8 li-bullet-0"><span class="c3"><a class="c9" href="#id.in9dk2kqjccl">Fine-scale recombination patterns differ between chimpanzees and humans</a></span></li><li class="c6 c8 li-bullet-0"><span class="c3"><a class="c9" href="#id.3mxv7b2r5r59">Absence of the TAP2 human recombination hotspot in chimpanzees</a></span></li><li class="c6 c8 li-bullet-0"><span class="c3"><a class="c9" href="#id.eoj2niyupk0l">Insights into recombination from patterns of linkage disequilibrium in humans</a></span></li><li class="c6 c8 li-bullet-0"><span class="c3"><a class="c9" href="#id.q8fn0swd3row">Mutation and recombination are associated processes in humans</a></span></li><li class="c6 c8 li-bullet-0"><span class="c3"><a class="c9" href="#id.7rwkod2jlxh8">Evidence for population growth in humans is confounded by fine-scale population structure</a></span></li><li class="c6 c8 li-bullet-0"><span class="c3"><a class="c9" href="#id.pu8dnqn5bjyc">How intron splicing affects the insertion and deletion profile in </a></span><span class="c3 c12"><a class="c9" href="#id.pu8dnqn5bjyc">Drosophila melanogaster</a></span></li><li class="c6 c8 li-bullet-0"><span class="c3"><a class="c9" href="#id.u4o7elcudbgv">On the evolution of polygamy: a theoretical examination of the polygamy threshold model</a></span></li><li class="c6 c8 li-bullet-0"><span class="c3"><a class="c9" href="#id.3gw7gxdt5knp">The maintenance of single-locus polymorphism V. Sex-dependent viabilities</a></span></li></ol><p class="c2"><span class="c0"></span></p><p class="c6"><span class="c1 c18"><a class="c9" href="https://www.google.com/url?q=https://susanptak.github.io&amp;sa=D&amp;source=editors&amp;ust=1701904632688092&amp;usg=AOvVaw0L44LA7vpGogm1cNq1Jzi9">Back to homepage</a></span></p><p class="c2"><span class="c0"></span></p><hr><p class="c2"><span class="c0"></span></p><a id="id.plu853aarphe"></a><p class="c6"><span class="c1">Bonobo genome compared with the chimpanzee and human genomes<br></span><span class="c7">Kay Pru&#776;fer, Kasper Munch, Ines Hellmann, Keiko Akagi, Jason R. Miller, Brian Walenz, Sergey Koren, Granger Sutton, Chinnappa Kodira, Roger Winer, James R. Knight, James C. Mullikin, Stephen J. Meader, Chris P. Ponting, Gerton Lunter, Saneyuki Higashino, Asger Hobolth, Julien Dutheil, Emre Karakoc&#807;, Can Alkan, Saba Sajjadian, Claudia Rita Catacchio, Mario Ventura, Tomas Marques-Bonet, Evan E. Eichler, Claudine Andre, Rebeca Atencia, Lawrence Mugisha, J&ouml;rg Junhold, Nick Patterson, Michael Siebauer, Jeffrey M. Good, Anne Fischer, Susan E. Ptak, Michael Lachmann, David E. Symer, Thomas Mailund, Mikkel H. Schierup, Aida M. Andre&#769;s, Janet Kelso</span><span class="c7">&nbsp;</span><span class="c7">&amp; Svante Pa&#776;a&#776;bo<br></span><span class="c0">Nature 486: 527-531 (2012)</span></p><p class="c6"><span class="c7">Two African apes are the closest living relatives of humans: the chimpanzee (</span><span class="c7 c12">Pan troglodytes</span><span class="c7">) and the bonobo (</span><span class="c7 c12">Pan paniscus</span><span class="c0">). Although they are similar in many respects, bonobos and chim- panzees differ strikingly in key social and sexual behaviours, and for some of these traits they show more similarity with humans than with each other. Here we report the sequencing and assembly of the bonobo genome to study its evolutionary relationship with the chimpanzee and human genomes. We find that more than three per cent of the human genome is more closely related to either the bonobo or the chimpanzee genome than these are to each other. These regions allow various aspects of the ancestry of the two ape species to be reconstructed. In addition, many of the regions that overlap genes may eventually help us understand the genetic basis of phenotypes that humans share with one of the two apes to the exclusion of the other.</span></p><p class="c2"><span class="c0"></span></p><hr><p class="c2"><span class="c11 c1"></span></p><a id="id.g98eyemwl1o3"></a><p class="c6"><span class="c1">Bonobos fall within the genomic variation of chimpanzees<br></span><span class="c0">Anne Fischer, Kay Pru&#776;fer, Jeffrey M. Good, Michel Halbwax, Victor Wiebe, Claudine Andre&#769;, Rebeca Atencia, Lawrence Mugisha, Susan E. Ptak, Svante Pa&#776;a&#776;bo<br>PLoS ONE 6(6): e21605 (2011)</span></p><p class="c6"><span class="c0">To gain insight into the patterns of genetic variation and evolutionary relationships within and between bonobos and chimpanzees, we sequenced 150,000 base pairs of nuclear DNA divided among 15 autosomal regions as well as the complete mitochondrial genomes from 20 bonobos and 58 chimpanzees. Except for western chimpanzees, we found poor genetic separation of chimpanzees based on sample locality. In contrast, bonobos consistently cluster together but fall as a group within the variation of chimpanzees for many of the regions. Thus, while chimpanzees retain genomic variation that predates bonobo-chimpanzee speciation, extensive lineage sorting has occurred within bonobos such that much of their genome traces its ancestry back to a single common ancestor that postdates their origin as a group separate from chimpanzees.</span></p><p class="c2"><span class="c0"></span></p><hr><p class="c2"><span class="c0"></span></p><a id="id.lsv089d3trxa"></a><p class="c6 c16"><span class="c11 c1">Linkage disequilibrium extends across putative selected sites in FOXP2</span></p><p class="c6 c14"><span class="c0">Susan E. Ptak, Wolgang Enard, Victor Wiebe, Ines Hellmann, Johannes Krause, Michael Lachmann, Svante P&auml;&auml;bo</span></p><p class="c6"><span class="c0">&nbsp; Molecular Biology and Evolution 26(10): 2181-2184 (2009)</span></p><p class="c6"><span class="c0">&nbsp;</span></p><p class="c6 c10"><span class="c7">Polymorphism data in humans suggest that the gene encoding the transcription factor FOXP2, which influences speech and language development, has been subject to a selective sweep within the last 260,000 years. It has been proposed that one or both of two substitutions that occurred on the human evolutionary lineage and changed amino acids were the targets for selection. In apparent contradiction to this is the observation that these substitutions are present in Neandertals who diverged from humans maybe 300,000-400,000 years ago. We have collected polymorphism data upstream and downstream of the substitutions. Contrary to what is expected following a selective sweep, we find that the haplotypes extend across the two sites. We discuss possible explanations for these observations. One of them is that the selective sweep reflected in </span><span class="c7 c12">FOXP2</span><span class="c0">&nbsp;polymorphism data was not associated with the two amino acid substitutions.</span></p><p class="c2"><span class="c0"></span></p><hr><p class="c2"><span class="c11 c13"></span></p><a id="id.afc7sqbzs1o"></a><p class="c6 c14"><span class="c1 c11">Combining sperm typing and linkage disequilibrium analyses reveals differences in selective pressures or recombination rates across human populations. </span></p><p class="c6 c4"><span class="c0">Vanessa J. Clark, Susan E. Ptak, Irene Tiemann, Yudong Qian, Graham Coop, Anne C. Stone, Molly Przeworski, Norman Arnheim, Anna Di Rienzo </span></p><p class="c6 c4"><span class="c0">Genetics 175: 795&ndash;804 (2007)<br></span></p><p class="c6 c4"><span class="c0">A previous polymorphism survey of the type 2 diabetes gene CAPN10 identified a segment showing an excess of polymorphism levels in all population samples, coinciding with localized breakdown of linkage disequilibrium (LD) in a sample of Hausa from Cameroon, but not in non-African samples. This raised the possibility that a recombination hotspot is present in all populations and we had insufficient power to detect it in the non-African data. To test this possibility, we estimated the crossover rate by sperm typing in five non-African men; these estimates were consistent with the LD decay in the non-African, but not in the Hausa data. Moreover, resequencing the orthologous region in a sample of Western chimpanzees did not show either an excess of polymorphism level or rapid LD decay, suggesting that the processes underlying the patterns observed in humans operated only on the human lineage. These results suggest that a hotspot of recombination has recently arisen in humans and has reached higher frequency in the Hausa than in non Africans, or that there is no elevation in crossover rate in any human population, and the observed variation results from long-standing balancing selection.</span></p><p class="c2"><span class="c0"></span></p><hr><p class="c2"><span class="c0"></span></p><a id="id.vnwo1z9n6imo"></a><p class="c6 c4"><span class="c11 c1">Analysis of one million base pairs of Neanderthal DNA.</span></p><p class="c6 c4"><span class="c0">Richard E. Green, Johannes Krause, Susan E. Ptak, AdrianW. Briggs, Michael T. Ronan, Jan F. Simons, Lei Du, Michael Egholm, Jonathan M. Rothberg, Maja Paunovic, Svante P&auml;&auml;bo<br>Nature 444: 330-336 (2006)</span></p><p class="c6 c4"><span class="c0">&nbsp;</span></p><p class="c6 c4"><span class="c0">Neanderthals are the extinct hominid group most closely related to contemporary humans, so their genome offers a unique opportunity to identify genetic changes specific to anatomically fully modern humans.We have identified a 38,000-year-old Neanderthal fossil that is exceptionally free of contamination from modern human DNA. Direct high-throughput sequencing of a DNA extract from this fossil has thus far yielded over one million base pairs of hominoid nuclear DNA sequences. Comparison with the human and chimpanzee genomes reveals that modern human and Neanderthal DNA sequences diverged on average about 500,000 years ago. Existing technology and fossil resources are now sufficient to initiate a Neanderthal genome-sequencing effort.</span></p><p class="c2"><span class="c0"></span></p><hr><p class="c2 c4"><span class="c0"></span></p><a id="id.ly189d50cg7p"></a><p class="c6 c4"><span class="c1">Why do human diversity levels vary at a megabase scale?<br></span><span class="c0">Ines Hellmann, Kay Pr&#363;fer, Hongkai Ji, Michael C. Zody, Svante P&auml;&auml;bo, Susan E. Ptak<br>Genome Research 15: 1222-1231 (2005)</span></p><p class="c6 c4"><span class="c0">Levels of diversity vary across the human genome. This variation is caused by two forces: differences in mutation rates and the differential impact of natural selection. Pertinent to the question of the relative importance of these two forces is the observation that both diversity within species and interspecies divergence increase with recombination rates. This suggests that mutation and recombination are either directly coupled or linked through some third factor. &nbsp;Here, we test these possibilities using the recently generated sequence of the chimpanzee genome and new estimates of human diversity. We find that measures of GC and CpG content, simple-repeat structures, as well as the distance from centromeres and the telomeres predict diversity as well as divergence. After controlling for these factors, large-scale recombination rates measured from pedigrees are still significant predictors of human diversity and human-chimpanzee divergence. Furthermore, the correlation between human diversity and recombination remains significant even after controlling for human-chimpanzee divergence. Two plausible and non-mutually exclusive explanations are, first, that natural selection has shaped the patterns of diversity seen in humans, and second, that recombination rates across the genome have changed since humans and chimpanzees shared a common ancestor, so that current recombination rates are a better predictor of diversity than divergence. &nbsp;Because there are indications that recombination may have changed rapidly during human evolution, we favor the latter explanation.</span></p><p class="c2 c4"><span class="c0"></span></p><hr><p class="c2"><span class="c0"></span></p><a id="id.in9dk2kqjccl"></a><p class="c6 c4"><span class="c1">Fine-scale recombination patterns differ between chimpanzees and humans<br></span><span class="c0">Susan E. Ptak, David A. Hinds, Kathrin Koehler, Birgit Nickel, Nila Patil, Dennis G. Ballinger, Molly Przeworski, Kelly A. Frazer, Svante P&auml;&auml;bo<br>Nature Genetics 37: 429-434 (2005)</span></p><p class="c6 c4"><span class="c0">Recombination rates seem to vary extensively along the human genome. &nbsp;Pedigree analysis suggests that rates vary by an order of magnitude when measured at the megabase scale, and at a finer scale, sperm typing studies point to the existence of recombination hotspots. These are short regions (1-2 kb) in which recombination rates are 10-1000 times higher than the background rate. Less is known about how recombination rates change over time. Here we determine to what degree recombination rates are conserved among closely related species by estimating recombination rates from 14 Mb of linkage disequilibrium data in central chimpanzee and human populations. The results suggest that recombination hotspots are not conserved between the two species and that recombination rates in larger (50 kb) genomic regions are only weakly conserved. Therefore, the recombination landscape has changed markedly between the two species.</span></p><p class="c2 c4"><span class="c0"></span></p><hr><p class="c2"><span class="c0"></span></p><a id="id.3mxv7b2r5r59"></a><p class="c6 c4"><span class="c1">Absence of the TAP2 human recombination hotspot in chimpanzees<br></span><span class="c0">Susan E. Ptak, Amy D. Roeder, Matthew Stephens, Yoav Gilad, Svante P&auml;&auml;bo, Molly Przeworski<br>PloS 2: 849-855 (2004)</span></p><p class="c6 c4"><span class="c0">Recent experiments using sperm typing have demonstrated that, in several regions of the human genome, recombination does not occur uniformly but instead is concentrated in &lsquo;&lsquo;hotspots&rsquo;&rsquo; of 1&ndash;2 kb. Moreover, the crossover asymmetry observed in a subset of these has led to the suggestion that hotspots may be short-lived on an evolutionary time scale. To test this possibility, we focused on a region known to contain a recombination hotspot in humans, TAP2, and asked whether chimpanzees, the closest living evolutionary relatives of humans, harbor a hotspot in a similar location. Specifically, we used a new statistical approach to estimate recombination rate variation from patterns of linkage disequilibrium in a sample of 24 western chimpanzees (Pan troglodytes verus). This method has been shown to produce reliable results on simulated data and on human data from the TAP2 region. Strikingly, however, it finds very little support for recombination rate variation at TAP2 in the western chimpanzee data. Moreover, simulations suggest that there should be stronger support if there were a hotspot similar to the one characterized in humans. Thus, it appears that the human TAP2 recombination hotspot is not shared by western chimpanzees. These findings demonstrate that fine-scale recombination rates can change between very closely related species and raise the possibility that rates differ among human populations, with important implications for linkage-disequilibrium based association studies.</span></p><p class="c2 c4"><span class="c0"></span></p><hr><p class="c2"><span class="c0"></span></p><a id="id.eoj2niyupk0l"></a><p class="c6 c4"><span class="c1">Insights into recombination from patterns of linkage disequilibrium in humans<br></span><span class="c0">Susan E. Ptak, Kristian Voelpel, Molly Przeworski<br>Genetics 167: 387-397 (2004) </span></p><p class="c6 c4"><span class="c0">An ability to predict levels of linkage disequilibrium (LD) between linked markers would facilitate the design of association studies and help to distinguish between evolutionary models. Unfortunately, levels of LD depend crucially on rates of recombination, a parameter that is difficult to measure. In humans, rates of genetic exchange between markers megabases apart can be estimated from a comparison of genetic and physical maps; these large-scale estimates can then be interpolated to predict LD at smaller (&quot;local&quot;) scales. However, if there is extensive small-scale heterogeneity, as has been recently proposed, local rates of recombination could differ substantially from those averaged over much larger distances. We test this hypothesis by estimating local recombination rates indirectly from patterns of LD in 84 genomic regions surveyed by the SeattleSNPs project in a sample of individuals of European descent and of African-Americans. We find that LD-based estimates are significantly positively correlated with map-based estimates. This implies that large-scale, average rates are informative about local rates of recombination. Conversely, although LD-based estimates are based on a number of simplifying assumptions, it appears that they capture considerable information about the underlying recombination rate, or at least about the ordering of regions by recombination rate. Using LD-based estimators, we also find evidence for homologous gene conversion in patterns of polymorphism. However, as we demonstrate by simulation, inferences about gene conversion are unreliable, even with extensive data from homogeneous regions of the genome, and are confounded by genotyping error.</span></p><p class="c2 c4"><span class="c0"></span></p><hr><p class="c2"><span class="c0"></span></p><a id="id.q8fn0swd3row"></a><p class="c6"><span class="c1">Mutation and recombination are associated processes in humans. <br></span><span class="c0">Ines Hellmann, Ingo Ebersberger, Susan E. Ptak, Svante P&auml;&auml;bo, Molly Przeworski <br>American Journal of Human Genetics 72: 1527-1535 (2003)</span></p><p class="c6"><span class="c0">One of the most striking findings to emerge from the study of genomic patterns of variation is that regions with lower recombination rates tend to have lower levels of intraspecific diversity but not of interspecies divergence. This uncoupling of variation within and between species has been widely interpreted as evidence that natural selection shapes patterns of genetic variability genomewide. &nbsp;We revisited the relationship between diversity, divergence and recombination in humans, using data from closely related species and better estimates of recombination rates than previously available. We show that regions that experience less recombination have reduced divergence to chimpanzee and to baboon, as well as lower levels of diversity. This observation suggests that mutation and recombination are associated processes in humans, so that the positive correlation between diversity and recombination may have a purely neutral explanation. &nbsp;Consistent with this hypothesis, diversity levels no longer increase significantly with recombination rates after correction for divergence to the chimpanzee.</span></p><p class="c2"><span class="c0"></span></p><hr><p class="c2"><span class="c0"></span></p><a id="id.7rwkod2jlxh8"></a><p class="c6 c4"><span class="c1">Evidence for population growth in humans is confounded by fine-scale population structure<br></span><span class="c0">Susan E. Ptak and Molly Przeworski<br>Trends in Genetics 18: 559-563 (2002)</span></p><p class="c6 c4"><span class="c0">&nbsp;</span></p><p class="c6 c4"><span class="c0">Although many studies have reported human polymorphism data, there has been no analysis of the effect of sampling design on the patterns of variability recovered. &nbsp;Here, we consider which factors affect a summary of the allele frequency spectrum. The most important variable to emerge from our analysis is the number of ethnicities sampled: studies that sequence individuals from more ethnicities recover more rare alleles. These observations are consistent with fine-scale geographic differentiation as well as population growth. They suggest that the geographic sampling strategy should be considered carefully, especially when the aim is to infer the demographic history of humans.</span></p><p class="c2 c4"><span class="c0"></span></p><hr><p class="c2 c4"><span class="c0"></span></p><a id="id.pu8dnqn5bjyc"></a><p class="c6 c4"><span class="c1">How intron splicing affects the insertion and deletion profile in </span><span class="c1 c12">Drosophila melanogaster<br></span><span class="c0">Susan E. Ptak and Dmitri Petrov <br>Genetics 162: 1233-1244 (2002)</span></p><p class="c6 c4"><span class="c7">Studies of &quot;dead-on-arrival&quot; transposable elements in </span><span class="c7 c12">Drosophila melanogaster</span><span class="c7">&nbsp;found that deletions outnumber &nbsp;insertions ~ 8:1 with a median size for deletions of ~10 bp. These results are consistent with the deletion and insertion profiles found in most other Drosophila pseudogenes. &nbsp;In contrast, a recent study of </span><span class="c7 c12">D. melanogaster</span><span class="c0">&nbsp;introns found a deletion/insertion ratio of 1.35:1, with 84% of deletions being shorter than 10 bp. &nbsp;This discrepancy could be explained if deletions, especially long deletions, are more frequently strongly deleterious than insertions and are eliminated disproportionately from intron sequences. &nbsp;To test this possibility, we use analysis and simulations to examine how deletions and insertions of different lengths affect different components of splicing and determine the distribution of deletions and insertions that preserve the original exons. &nbsp;We find that, consistent with our predictions, longer deletions affect splicing at a much higher rate compared to insertions and short deletions. &nbsp;We also explore other potential constraints in introns and show that most of these also disproportionately affect large deletions. &nbsp;Altogether we demonstrate that constraints in introns may explain much of the difference in the pattern of deletions and insertions observed in Drosophila introns and pseudogenes.</span></p><p class="c2 c4"><span class="c0"></span></p><hr><p class="c2 c4"><span class="c0"></span></p><a id="id.u4o7elcudbgv"></a><p class="c6 c4"><span class="c1">On the evolution of polygamy: a theoretical examination of the polygamy threshold model <br></span><span class="c7">Susan E. Ptak and Michael Lachmann <br>Behavioral Ecology 14: 201-211 (2002)<br></span><span class="c5"><a class="c9" href="https://www.google.com/url?q=http://www.santafe.edu/sfi/publications/wpabstract/200101001&amp;sa=D&amp;source=editors&amp;ust=1701904632694460&amp;usg=AOvVaw2GIbwo-jF0D-pAf2_wUaL8">Santa Fe Institute Working Paper 01-01-001</a></span></p><p class="c6 c4"><span class="c0">&nbsp;</span></p><p class="c6 c4"><span class="c0">The polygany threshold model states that if costs incurred are less than the benefits gained from &nbsp;mating polygynously in terms of male breeding-situation quality, then polygyny is favored and could evolve. &nbsp;We constructed mathematical models and computer simulations to evaluate this hypothesis. &nbsp;In the basic model, there is a single locus with two alleles, which regulates whether the female is receptive to polygyny. &nbsp;There are two breeding situations of differing quality on which males randomly assort. &nbsp;Females then select a mate based on the associated breeding situation and whether the male already has mates. &nbsp;This basic model is extended mathematically to include cost for the initial female of a male with multiple mates and again to include gene expression in males. The computer simulations extend the basic model to multiple loci and alleles, and to multiple breeding situations. &nbsp;The results presented here suggest that the polygyny threshold model is valid in a population genetic context: if the fitness of females that actually mated polygynously is greater than the fitness of monogamous females on poorer breeding situations, polygyny evolves. &nbsp;However, this approach reveals interesting dynamics not apparent from the verbal model. &nbsp;If the trait is expressed in males and females, then polygyny can evolve even if females mating polygynously have a lower fitness than females mating monogamously. &nbsp;In the multiple breeding-situations model, the polygyny allele increases to some equilibrium value above which it experiences no selection. &nbsp;Surprisingly, as the cost to polygyny increases, the equilibrium value of the polygyny allele also increases. The difference between this evolutionary model and the ideal free distribution is discussed.</span></p><p class="c2 c4"><span class="c0"></span></p><hr><p class="c2"><span class="c0"></span></p><a id="id.3gw7gxdt5knp"></a><p class="c6 c4"><span class="c1">The maintenance of single-locus polymorphism. &nbsp;V. &nbsp;Sex-dependent viabilities.<br></span><span class="c0">R. William Marks and Susan E. Ptak <br>Selection 1: &nbsp;217-228 (2001)</span></p><p class="c4 c6"><span class="c0">Since natural selection requires variation to act, the amount of genetic variation in a given population is of central theoretical and experimental importance. &nbsp;This amount is estimated by the genetic variation present in current populations. &nbsp;Electrophoretic studies of natural populations reveal that ten to twenty percent of all loci have multiple alleles. &nbsp;These multi-allelic loci may be the result of the accumulation of unselected mutations (neutral theory) or of balancing selection (selectionist theory). &nbsp;The neutral theory views selection as primarily eliminating variation,whereas the selectionist theory views selection as often maintaining variation. &nbsp;Therefore, our view of selection and evolution depends upon the question of how multiple allelic systems evolve. &nbsp;To address whether balancing selection increases genetic variation, we examined Owen&#39;s model in which selection varies in the two sexes (sex-dependent model) and compared it to a model in which selection is constant across the two sexes (sex-independent model). We explored these models for a single multi-allelic locus to explore more generally the effect of balancing selection on the maintenance and construction of multi-allelic systems. &nbsp;To do so, we constructed two computer models. The &quot;fitness-space&quot;approach examines the proportion of all possible fitnesses capable of maintaining different sized allelic systems. &nbsp;The proportion of possible fitnesses is greater in balancing selection models. &nbsp;The&quot;constructionist&quot; approach examines the ease in reaching this portion of possible fitnesses. &nbsp;The ease in reaching fitnesses is less in balancing selection models. &nbsp;Thus, in contrast to previous hypotheses, our research suggests that balancing selection may not increase the amount of variation present in natural populations.</span></p><p class="c2"><span class="c0"></span></p><p class="c2 c4"><span class="c0"></span></p><p class="c2 c4"><span class="c0"></span></p><p class="c2 c4"><span class="c0"></span></p><p class="c2"><span class="c0"></span></p></body></html>